vmdb/app/models/metric/capture/openstack.rb

```diff
@@ -12,6 +12,14 @@ def self.diff_meter?(meters)
     meters.all? {|m| DIFF_METERS.include? m}
   end
 
+  def self.counter_sum_per_s_calculation(stats, intervals)
```

Comment:
I commented on the other PR, but should have done it here.

What is "per_s"...can we spell it out, so there's no guesswork?